tower-load-shed/src/lib.rs

```diff
@@ -0,0 +1,80 @@
+#![cfg_attr(test, deny(warnings))]
+#![deny(missing_debug_implementations)]
+#![deny(missing_docs)]
+
+//! tower-load-shed
+
+extern crate futures;
+extern crate tower_layer;
+extern crate tower_service;
+
+use futures::Poll;
+use tower_service::Service;
+
+pub mod error;
+mod future;
```

Comment:
This probably should be pub... it is pub everywhere else. Do we want to explicitly make response futures not nameable?

Reply:
I don't care too much either way. I don't think people should name this one, because it will be annoying if you change around the layers, but it's easy to mark public.

Reply:
I would either like to make this public or make all the other ones private. Which do you think is best?

Reply:
I made the re-export public.